aries-core/src/js/components/layouts/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Heading, Paragraph } from 'grommet';
+
+export const EmptyState = ({
+  actions,
+  align,
+  description,
+  icon,
+  level,
+  title,
+}) => {
+  return (
+    <Box gap="medium" align={align || 'center'} flex={false}>
+      {icon}
+      <Box align={align || 'center'} gap="xsmall">
+        <Heading margin="none" level={level}>
+          {title}
+        </Heading>
+        <Paragraph margin="none" textAlign={align || 'center'}>
```

Comment:
🤔  I'm wondering about the 'center' alignment of the textAlign...  I can't think of anywhere else where we do that. Should we always treat paragraphs as left justified?

@KennyAtHPE thoughts?

Reply:
Based on share-out discussion, Chris seemed comfortable with center alignment for this use case.

Reply:
Agree.